### DISCUSSION TITLE
BT controllers not connecting to HW2 FW DIY Installation N64

### DISCUSSION THREAD

**Discussion by Valkristh** [darthcloud/BlueRetro#501] [Q&A]
I will never finish thanking what you are doing and I like to test the new changes, that being said,
As the title says, I'm testing the fw in my internal N64 installation, tried with wiimote, wiimote + classic and WiiU Pro, non of these are connecting, i have the 4 data ports connected, before that the connection worked fine, if you need any more info I'm here
Greetings

**Comment by darthcloud** [comment]
Sound like you updated the FW drom an existing install.

HW2 as the name suggest is an entirely different installation type which requires a lot of new signal connection to work

You need to use HW1 FW most likely if you used to use 1.7.x and lower before.

**Reply by Valkristh** [reply]
I did it via usb on a pc, first reset the esp32 to factory and then installed it, or was that not enough and should i use a new esp32 and new connections?
